rdkcentral/Dobby
# Print log level in journald

### Description
When logging to journald, print the log level as well as the message.

Whilst we do use systemd priorities, this gets lost when the logs are dumped to a file. 

This allows QA to grep the dobby log for errors (`cat /opt/logs/sky-messages.log | grep ERR`) easily.

### Test Procedure
Ensure log messages in journald have suitable prefixes

```
ov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: started sd-bus event loop thread
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: starting sd-bus event loop
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: found cgroup 'cpuset' mounted @ '/sys/fs/cgroup/cpuset'
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: found cgroup 'freezer' mounted @ '/sys/fs/cgroup/freezer'
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: found cgroup 'blkio' mounted @ '/sys/fs/cgroup/blkio'
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: found cgroup 'memory' mounted @ '/sys/fs/cgroup/memory'
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: found cgroup 'cpuacct' mounted @ '/sys/fs/cgroup/cpu,cpuacct'
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: found cgroup 'devices' mounted @ '/sys/fs/cgroup/devices'
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: WRN: failed to get major number for driver 'hidraw'
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: WRN: failed to get major number of hidraw device
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: entered Ipc service change thread
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: entered Ipc service change thread
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: missing AI_PLATFORM_TYPE environment var, will set empty container platform env vars
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: Logging socket created at '/tmp/dobbyPty.sock'
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: entered Ipc service change thread
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: Dobby Logger socket monitoring thread started
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: starting watchdog timer with period 2500000
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: NFO: started SIGCHLD monitor thread
Nov 08 11:43:44 dobby-vagrant-focal DobbyDaemon[131167]: MIL: started Dobby daemon
```

### Type of Change

### Requires Bitbake Recipe changes?
